src/katsdpcalproc/pointing.py

```diff
@@ -152,7 +152,7 @@ def __init__(self, center, width, height):
         # Fix theta = 0 to ensure that elliptical beam contours line up with axes
         model.theta.fixed = True
         self._set_model(model)
-        self._fit = fitting.LMLSQFitter(calc_uncertainties=True)
+        self._fit = fitting.LevMarLSQFitter(calc_uncertainties=True)
```

Comment:
Do you want to add a "To do" comment here about updating the fitter in the future ?

Reply:
Great idea, done!